partialWaveFit/fitResult.cc

```diff
@@ -343,6 +342,14 @@ fitResult::fill(const fitResult& result)
 	_hasHessian             = result.hasHessian();
 }
 
+multibinCenterType fitResult::multibinCenter() const
+{
+	multibinCenterType multibinCenter;
+	for (binningMapType::const_iterator it = _binningMap.begin(); it != _binningMap.end(); ++it) {
+		multibinCenter[it->first] = 0.5 * (it->second.first + it->second.second);
+	}
+	return multibinCenter;
+}
```

Comment:
Does the mean value of the limits make sense for anything but the mass? For example, for tPrime the mean of the individual values and the mean of the limits will typically systematically deviate.